package.json

```diff
@@ -63,16 +59,17 @@
     "file-saver": "^2.0.2",
     "file-type": "^16.2.0",
     "fontsource-roboto": "^3.0.3",
+    "history": "4.10.1",
```

Comment:
Musi być taka
Inaczej `React-Router` przestaje działać
https://stackoverflow.com/a/66778409/7516134

Nie wiem z czym to dokładnie jest związane, ale z racji tego, że nie używamy go w dużej ilości miejsc nie zaglębiałem się w to